docs/dependency-injection-liftweb-scala.adoc

```diff
@@ -8,57 +8,45 @@
+choices about which combination of dependencies to vend in a given situation. Lift provides some additional
+features that complete the dependency injection puzzle.
 
-Although the cake pattern is helpful, it doesn't provide complete DI functionality.
-The cake pattern allows you to compose complex classes out of Scala traits, but the it's less helpful
-in terms of allowing you to make dynamic choices about which combination of cake to vend in a given situation.
-Lift provides extra features that complete the dependency injection puzzle.
+== Why Lift's Dependency Injection
 
-== Why Lift DI
-
-Many people come to Scala from a Java background, so it's only natural to want to use Guice/Spring,
-which do a great job in the java world. And we can https://groups.google.com/forum/#!topic/liftweb/_lleL2xpCFU[use those],
-but there's a better, more direct way that's less magical.
+Many people come to Scala from a Java background, so it is only natural to want to use Guice/Spring,
+which do a great job in the java world. While https://groups.google.com/forum/#!topic/liftweb/_lleL2xpCFU[those can be used],
+there is a better, more direct way that is less magical.
 
 Lift's philosophy is to keep things as simple as possible (and no simpler), and allow full control
 to the developers over how things work. To paraphrase David Pollak,
-"The abstraction turtles have to end somewhere" and  lift's DI features end those turtles quite
+"The abstraction turtles have to end somewhere" and  Lift's Dependency Injection features end those turtles quite
 early.
 
-== Lift DI and various dependency scopes (e.g. from Spring)
-
-Lift's DI allows all the
-https://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html#beans-factory-scopes[four most commonly used scopes] that
-are possible with spring:
-https://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html#beans-factory-scopes-singleton[singleton],
-https://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html#beans-factory-scopes-prototype[prototype], https://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html#beans-factory-scopes-request[request],
-and https://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html#beans-factory-scopes-session[session]. The others can be achieved
-relatively easily if you want.
-
 == Getting started
 
-Lift's DI is based on two main traits https://liftweb.net/api/26/api/index.html#net.liftweb.util.Injector[`Injector`]
-and https://liftweb.net/api/26/api/index.html#net.liftweb.util.Maker[`Maker`]. However, most of the time, you won't need to
-deal with them directly. The elements that you would use are:
+Lift's dependency injection is based on two main traits: https://liftweb.net/api/26/api/index.html#net.liftweb.util.Injector[`Injector`]
+and https://liftweb.net/api/26/api/index.html#net.liftweb.util.Maker[`Maker`]; however, most of the time,
```

Comment:
API links now point to Lift 3.1. About moving the latest release to `/latest`. Want to start doing something about it? Sounds like a good idea, and it's pretty standard.